datalake/src/main/java/com/sequenceiq/datalake/flow/SdxReactorFlowManager.java

```diff
@@ -116,6 +119,35 @@ public FlowIdentifier triggerDatalakeRuntimeUpgradeFlow(SdxCluster cluster, Stri
         }
     }
 
+    private boolean shouldSdxBackupBePerformed(SdxCluster cluster) {
+        boolean retVal = true;
+        String reason = null;
+        if (cluster.isRangerRazEnabled()) {
+            retVal = false;
+            reason = "RAZ is enabled";
+        } else if (isVersionOlderThan(cluster, "7.2.1")) {
+            retVal = false;
+            reason = "Unsupported runtime: " + cluster.getRuntime();
+        } else if (cluster.getCloudStorageFileSystemType().isGcs()) {
```

Comment:
**suggestion:** I think it would be worth calling out that we're using the storage file system type as a proxy for the cloud provider. Maybe in a javadoc comment on the method.

Reply:
done.